source/endpoint-reference/endpoint-folders.rst

```diff
@@ -201,10 +197,30 @@ Trash
 
 ``/datasets/{id}/folders/trash/``
 
-A special folder, the folders' trash exists to store folders after they get
-deleted.
+The trash folder is a special-purpose folder that lives outside the
+regular variable folders tree.
+
+You can use the trash folder to "soft-delete" variables and folders.
+
+Subfolders and variables inside the trash folder are only visible to
+dataset editors.
+
+Performing a DELETE request to the trash endpoint will empty all items
+from the trash, hard-deleting the folders, variables, and related data.
+
+Items in the trash folder may also be automatically hard-deleted after
+24-48 hours.
+
+Hidden
+------
+
+``/datasets/{id}/folders/hidden/``
 
-Performing a DELETE request to the trash endpoint will empty all items from
-the trash, hard-deleting all folders, variables, and related data.
+The hidden folder is a special-purpose folder that lives outside the
+regular variable folders tree.
 
+Variables and subfolders that should only be visible to dataset editors
+should be placed in the hidden folder.
 
+Hidden folder membership will eventually replace the deprecated
```

Comment:
Should this go as an aside? or warning?

Reply:
During the `discarded` deprecation task (not in scope here), I had planned on updating the docs that show how to make something `discarded` to instead show them how to move something to the hidden folder, and I'll add warnings then.